src/components/transactions/RejectTxButton/index.tsx

```diff
@@ -42,7 +44,7 @@ const RejectTxButton = ({
     <>
       <Track {...TX_LIST_EVENTS.REJECT}>
         {compact ? (
-          <Tooltip title="Reject" arrow placement="top">
+          <Tooltip title="Replace" arrow placement="top">
             <span>
               <IconButton onClick={onClick} color="error" size="small" disabled={isDisabled}>
                 <SvgIcon component={ErrorIcon} inheritViewBox fontSize="small" />
```

Comment:
I feel like the `ErrorIcon` does not capture the replacement feature as much as it did the rejection.

Reply:
@johannesmoormann mentioned that we will work on these designs some more so I think we can revisit this at a later stage.

![image](https://user-images.githubusercontent.com/20442784/199946150-6e3aeeae-c8ee-475d-aa96-649cfc21eb71.png)